Gems/EMotionFX/Code/CMakeLists.txt

```diff
@@ -111,6 +111,10 @@ if (PAL_TRAIT_BUILD_HOST_TOOLS)
             PUBLIC
                 EMFX_EMSTUDIOLYEMBEDDED
                 EMOTIONFXANIMATION_EDITOR
+        RUNTIME_DEPENDENCIES
+            AZ::SceneCore
+            AZ::SceneData
+            AZ::SceneUI
```

Comment:
Actually the SceneCore SceneData and SceneUI modules aren't gems and will not automatically load through the gem system, so this only causes those modules to build as a dependency of EMotionFX.Editor.Static target

The SceneProcessing gem is what actually properly initializes the Scene API modules
It already adds itself as a dependency of the Editor and AssetProcessor in it's CMakeLIsts.txt

The actually issue is that the SceneProcessing gem doesn't have a dependency on the AZ::SceneUI module.

So the actual fix is to update that gem CMakeLists.txt with an a runtime dependency on AZ::SceneUI

Reply:
Removed the runtime dependencies from the EmotionFX gem, added AZ::SceneUI as a runtime dependency of the SceneProcessing::Editor module.